registration-processor/registration-processor-common-camel-bridge/src/main/resources/bootstrap.properties

```diff
@@ -1,6 +1,6 @@
-spring.cloud.config.uri=localhost
+spring.cloud.config.uri=http://localhost:51000/config

```

Comment:
why is camel bootstrap changes committed ?

Reply:
reverted,it was added by mistake while testing